src/applications/avs/utils/medications/index.js

```diff
@@ -1,11 +1,27 @@
+import { MEDICATION_SOURCES } from '../constants';
+
 const filterMedicationsByType = (medications, type) => {
   return medications.filter(medication => medication.prescriptionType === type);
 };
 
+const addMedicationSource = (medications, source) => {
+  return medications.map(medication => {
+    const medicationWithSource = medication;
+    medicationWithSource.medicationSource = source;
+    return medicationWithSource;
```

Comment:
This feels clunky, but directly modifying `medication` triggers a `no-param-reassign` linting error.